libs/common/NetworkBuffers.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Extensions.Logging;
+
+namespace Garnet.common
+{
+    /// <summary>
+    /// Create a NetworkBuffers instance
+    /// </summary>
+    public struct NetworkBuffers
```

Comment:
This struct does not do much, can create challenges with single-copy as it is a struct with value semantics, and generally pervades across the codebase. The main this it does is to use two allocation sizes, which can be maintained by callers or within LFBP itself.